examples/basic/bolt_pretension.py

```diff
@@ -525,21 +525,7 @@ def update(frame):
 # Project tree
 # ~~~~~~~~~~~~
 
-
-def print_tree(node, indentation=""):
-    print(f"{indentation}├── {node.Name}")
-
-    if (
-        hasattr(node, "Children")
-        and node.Children is not None
-        and node.Children.Count > 0
-    ):
-        for child in node.Children:
-            print_tree(child, indentation + "|  ")
-
-
-root_node = DataModel.Project
-print_tree(root_node)
+app.print_tree(DataModel.Project)
```

Comment:
minor - I think print_tree should print the project by default

Reply:
Created a new issue in PyMechanical for this